### PR TITLE
feat: migrate to new monv2 action api (part 1: add new params)

### PR DESCRIPTION
**Pull request by obs-gh-owengoebel**

### CODE REVIEW

File: client/internal/meta/schema/monitorv2.graphql [hunk 1]
```diff
@@ -149,7 +160,7 @@ type MonitorV2 implements WorkspaceObject & AuditedObject & FolderObject @goMode
     A longer description of the monitor.
     This can include details like how to resolve the issue, links to runbooks, etc.
     """
-    comment: String
+    comment: String @deprecated(reason:"Use the description field over the comment field")
```
Comment (vikramraman) [inline client/internal/meta/schema/monitorv2.graphql:163]: Can we remove this. It was removed in an earlier commit, but looks like the schema copy brought it back.

File: docs/data-sources/monitor_v2.md [hunk 2]
```diff
@@ -184,7 +186,7 @@ Read-Only:
 - `compare_fn` (String) the type of comparison (greater, less, equal, etc.)
 - `value_bool` (List of Boolean) list of size <=1 consisting of a boolean value.
 - `value_duration` (List of Boolean) list of size <=1 consisting of a duration value.
-- `value_float64` (List of Number) list of size <=1 consisting of a float value.
+- `value_float64` (List of Number) list of size <=1 consisting of a float value
```
Comment (vikramraman) [inline docs/data-sources/monitor_v2.md:189]: nit: add back the period for consistency with the other fields here

File: docs/data-sources/monitor_v2_action.md [hunk 3]
```diff
@@ -23,50 +23,23 @@ description: |-
 ### Read-Only
 
 - `description` (String)
-- `destination` (Block List) (see [below for nested schema](#nestedblock--destination))
+- `destination` (String)
```
Comment (vikramraman) [inline docs/data-sources/monitor_v2_action.md:26]: I guess this can be removed in a follow up PR?

File: docs/resources/monitor_v2.md [hunk 4]
```diff
@@ -83,8 +83,8 @@ Required:
 Optional:
 
 - `value_bool` (List of Boolean) list of size <=1 consisting of a boolean value.
-- `value_duration` (List of Boolean) list of size <=1 consisting of a duration value.
-- `value_float64` (List of Number) list of size <=1 consisting of a float value.
+- `value_duration` (List of Number) list of size <=1 consisting of a duration value.
+- `value_float64` (List of Number) list of size <=1 consisting of a float value
```
Comment (vikramraman) [inline docs/resources/monitor_v2.md:87]: Similar nit

**Review by vikramraman — CHANGES_REQUESTED** [review]

**Comment by vikramraman** [comment]
Also the tests are failing with:
```
error=
  | Error running pre-apply refresh: exit status 1
  | 
  | Error: Unsupported block type
  | 
  |   on terraform_plugin_test.tf line 63, in resource "observe_monitor_v2_action" "act":
  |   63: \t\t\t\t\t\tdestination {
  | 
  | Blocks of type "destination" are not expected here. Did you mean to define
  | argument "destination"? If so, use the equals sign to assign it a value.
  
```

File: docs/data-sources/monitor_v2_action.md (hunk 3 above)
Comment (obs-gh-owengoebel) [reply docs/data-sources/monitor_v2_action.md:26]: Yeah, we discussed yesterday that I was going to submit a PR that phases out the dest API calls

File: client/internal/meta/schema/monitorv2.graphql (hunk 1 above)
Comment (obs-gh-owengoebel) [reply client/internal/meta/schema/monitorv2.graphql:163]: I have omitted it from the tf schema. It is fine for it to be in gql schema though.

**Review by vikramraman — APPROVED** [review]

**Pull request merged**